rgw/v2/tests/s3_swift/multisite_configs/test_Mbuckets_with_Nobjects_multipart_haproxy.yaml

```diff
@@ -0,0 +1,24 @@
+# upload type: non multipart
+# script: test_Mbuckets_with_Nobjects.py
+config:
+  haprocy: true
```

Comment:
Please make a correction to variable name "haproxy"

Reply:
done @ckulal